specification/netapp/resource-manager/Microsoft.NetApp/stable/2019-07-01/netapp.json

```diff
@@ -0,0 +1,2355 @@
+          "description": "The category this metric specification belong to, could be Capacity."
+        },
+        "resourceIdDimensionNameOverride": {
+          "type": "string",
+          "description": "Account Resource Id."
+        }
+      }
+    },
+    "Dimension": {
+      "description": "Dimension of blobs, possibly be blob type or access tier.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Display name of dimension."
+        },
+        "displayName": {
+          "type": "string",
+          "description": "Display name of dimension."
+        }
+      }
+    },
+    "ResourceNameAvailability": {
+      "description": "Information regarding availability of a resource name.",
+      "type": "object",
+      "properties": {
+        "isAvailable": {
+          "description": "<code>true</code> indicates name is valid and available. <code>false</code> indicates the name is invalid, unavailable, or both.",
+          "type": "boolean"
+        },
+        "reason": {
+          "description": "<code>Invalid</code> indicates the name provided does not match Azure App Service naming requirements. <code>AlreadyExists</code> indicates that the name is already in use and is therefore unavailable.",
+          "enum": [
+            "Invalid",
+            "AlreadyExists"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "InAvailabilityReasonType",
+            "modelAsString": true
+          }
+        },
+        "message": {
+          "description": "If reason == invalid, provide the user with the reason why the given name is invalid, and provide the resource naming requirements so that the user can select a valid name. If reason == AlreadyExists, explain that resource name is already in use, and direct them to select a different name.",
+          "type": "string"
+        }
+      }
+    },
+    "ResourceNameAvailabilityRequest": {
+      "description": "Resource name availability request content.",
+      "required": [
+        "name",
+        "type",
+        "resourceGroup"
+      ],
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Resource name to verify.",
+          "type": "string"
+        },
+        "type": {
+          "description": "Resource type used for verification.",
+          "enum": [
+            "Microsoft.NetApp/netAppAccounts",
+            "Microsoft.NetApp/netAppAccounts/capacityPools",
+            "Microsoft.NetApp/netAppAccounts/capacityPools/volumes",
+            "Microsoft.NetApp/netAppAccounts/capacityPools/volumes/snapshots"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "CheckNameResourceTypes",
+            "modelAsString": true
+          }
+        },
+        "resourceGroup": {
+          "description": "Resource group name.",
+          "type": "string"
+        }
+      }
+    },
+    "netAppAccountList": {
```

Comment:
Object definitions should be PascalCased.  Comment applies to multiple types.

Reply:
ok. Note that this is the same in earlier versions. Can it therefore be delayed until the next version since this swagger is already now overdue?

Reply:
@leonardbf , that makes sense since it is already there. Cab we work items to address these comments on a next release ?

Reply:
I see.  If there are previous releases with this pattern then let's just leave it as-is in order to not introduce any breaking changes.  Most code generators will automatically PascalCase the type names anyways.